x-pack/plugin/async-search/src/main/java/org/elasticsearch/xpack/search/AsyncSearchTask.java

```diff
@@ -381,6 +385,15 @@ public static AsyncStatusResponse getStatusResponse(AsyncSearchTask asyncTask) {
 
     @Override
     public void close() {
+        if (logger.isDebugEnabled()) {
+            logger.debug(
+                "AsyncSearchTask.close(): byThread={}, asyncId={}, taskId={}, hasCompleted={}",
```

Comment:
Is the thread name enough to help here? Would we want a bit of the stack trace ?